server/src/main/java/org/opensearch/action/search/SearchTask.java

```diff
@@ -49,10 +49,11 @@
  * @opensearch.api
  */
 @PublicApi(since = "1.0.0")
-public class SearchTask extends CancellableTask implements SearchBackpressureTask {
+public class SearchTask extends QueryGroupTask implements SearchBackpressureTask {
```

Comment:
Stub, will be introduced into main in this PR - https://github.com/opensearch-project/OpenSearch/pull/14708/files#diff-cb91ab0d31fd0988ed76dcac2d6c151523f0c7b904e42316956e12bf2c217899R53